test/system/admin/tag_test.rb

```diff
@@ -15,7 +15,7 @@ class AdminTagTest < ApplicationSystemTestCase
 
     @partner = @partner_admin.partners.first
     @partner_two = create :ashton_partner
-    @tag = create :tag
+    @tag = create(:tag, name: 'Hulme 1 Tag')
```

Comment:
i think this is an unhelpful name for a tag here - its unclear to me if this is a category or place we are testing?

Reply:
The test breaks without the ' Tag' postfix. At the moment we aren't testing anything like that as they don't exist, as per your instructions

Reply:
Sure - just saying there should never be a tag named after a place - so maybe call it `Arts and Crafts Tag` or something for now or it'll get confusing!